content/early-years-teaching-training.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Early years teacher training"
+image: false
+backlink: "../"
+keywords:
+ - EYITT
+ - Early
+ - Years
+ - Early Years
+ - EYTS
+ - Early Years Teacher Status
+ - Under 5
+
+calls_to_action:
+  EY-undergraduate:
+    name: feature_table
+    arguments:
+      - Cost: "Up to £9250"
+        Salary: "No"
+        Funding available: "Yes"
+        Length of training: "3 or 4 years"
+        Qualification: "Early Years Teacher Status"
+      - "Undergraduate early years teacher training"
+  EY-graduate:
+    name: feature_table
+    arguments:
+      - Cost: "£7000"
+        Salary: "No"
+        Funding available: "Yes"
+        Length of training: "Typically 1 academic year"
+        Qualification: "Early Years Teacher Status"
+      - "Graduate early years teacher training"     
+  EY-graduate-employment:
+    name: feature_table
+    arguments:
+      - Cost: "£7000"
+        Salary: "Provided by employer"
+        Funding available: "Yes"
+        Length of training: "Typically 1 academic year"
+        Qualification: "Early Years Teacher Status"
+      - "Graduate employment-based early years teacher training"
+  EY-self-funded:
+    name: feature_table
+    arguments:
+      - Cost: "Self-funded"
+        Salary: "No"
+        Funding available: "No"
+        Length of training: "3 months"
+        Qualification: "Early Years Teacher Status"
+      - "Graduate Assessment-Only training"
+---
+
+## Becoming an early years teacher
+ 
+As an early years teacher, you will work with young children from birth up to age five. Your input will have a lasting and positive effect on children’s development and wellbeing.
+ 
+By training to deliver the Early Years Foundation Stage, you’ll learn how to use your knowledge and skills to help young children enjoy high standards of teaching. These early years are critical for young children’s development, and you’ll be opening their minds to new ideas every day.
+ 
+###How to become an early years teacher
+ 
+To qualify, you’ll need Early Years Teacher Status, which is different from Qualified Teacher Status (the equivalent for teaching children aged three to 18). Your training will enable you to meet the [Teachers’ Standards (Early Years)](https://www.gov.uk/government/publications/early-years-teachers-standards).
+ 
+To start early years teacher training, you’ll need these qualifications (or be able to deomnstrate an equivalent standard):
+
+* Maths grade 4 (C)
+* English grade 4 (C)
+* Science grade 4 (C)
+
+Graduate training also requires a Bachelor’s degree with honours, class 2:2 or higher.
```

Comment:
```suggestion
Graduate training also requires a 2:2 bachelor’s degree or higher.
```

Reply:
We're calling this 'graduate training' here, but should it actually be 'postgraduate training', given they've already got their undergrad degree? On Find, we call these courses 'postgraduate teacher training'.